server/views/applications/pages/check-your-answers/check-your-answers.njk

```diff
@@ -22,90 +22,87 @@
 
 {% block content %}
   <div class="govuk-grid-row">
-    <div class="govuk-width-container">
-
-      {{ showErrorSummary(errorSummary) }}
-
-      {{ applicationSummary(page.applicationSummary()) }}
+				<div class="govuk-grid-column-full">
```

Comment:
Minor: might be worth just fixing the formatting in this file a little? Otherwise looks good